canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -74,27 +78,53 @@ class CheckboxsetControl extends React.Component {
 	}
 
 	render() {
+
+		const hidden = this.props.state === STATES.HIDDEN;
+		const disabledTip = this.props.state === STATES.DISABLED;
```

Comment:
We already have a disabled value below.  Why not move that up here and use it?

Reply:
Used it.